rt_utils/ds_helper.py

```diff
@@ -80,7 +80,7 @@ def add_study_and_series_information(ds: FileDataset, series_data):
     ds.StudyDescription = getattr(reference_ds, "StudyDescription", "")
     ds.SeriesDescription = getattr(reference_ds, "SeriesDescription", "")
     ds.StudyInstanceUID = reference_ds.StudyInstanceUID
-    ds.SeriesInstanceUID = generate_uid()  # TODO: find out if random generation is ok
+    ds.SeriesInstanceUID = reference_ds.SeriesInstanceUID
```

Comment:
I believe your solution here is correct.   The new images should be part of the previous study.

Reply:
@hjmjohnson Thanks for having a look at this! Would you mind reviewing this so it can be merged in a later release?

Reply:
This looks good to me.